apps/arweave/src/ar_chain_stats.erl

```diff
@@ -0,0 +1,176 @@
+-module(ar_chain_stats).
+
+-behaviour(gen_server).
+
+-include_lib("arweave/include/ar.hrl").
+-include_lib("arweave/include/ar_chain_stats.hrl").
+-include_lib("eunit/include/eunit.hrl").
+
+-export([log_fork/2, get_forks/1]).
+
+-export([start_link/0, init/1, handle_call/3, handle_cast/2, handle_info/2, terminate/2]).
+
+%%%===================================================================
+%%% Public interface.
+%%%===================================================================
+start_link() ->
+    gen_server:start_link({local, ?MODULE}, ?MODULE, [], []).
+
+log_fork([], _ForkRootB) ->
+    %% No fork to log
+    ok;
+log_fork(Orphans, ForkRootB) ->
+    gen_server:cast(?MODULE, {log_fork, Orphans, ForkRootB, os:system_time(millisecond)}).
+
+%% @doc Returns all forks that have been logged since the given start time
+%% (system time in seconds)
+get_forks(StartTime) ->
+    gen_server:call(?MODULE, {get_forks, StartTime}).
+
+%%%===================================================================
+%%% Generic server callbacks.
+%%%===================================================================
+init([]) ->
+    process_flag(trap_exit, true),
+    Columns = [
+        {"default", []},
+        {"forks", []}
```

Comment:
Do we need to use a custom RocksDB column? The feature has a bug where the data in the non-default columns would not survive the first non-graceful shutdown (some problems with WAL population, perhaps).

Reply:
Oh. Ok. So I should create a separate rocksdb file for each column? My plan is to add a few more columns in the ar_chai_stats module to track some other data needed for separate bugs/requests

Reply:
I would do one db per table, like we have it e.g., in [ar_storage](https://github.com/ArweaveTeam/arweave/blob/master/apps/arweave/src/ar_storage.erl#L1030), do you see major downsides to this approach?

Reply:
Nope! I was just trying to follow conventions and wasn't sure whether ar_data_sync or ar_storage was the "right" way. I'll do one file per table!

Reply:
Updated: https://github.com/ArweaveTeam/arweave/pull/581/commits/b63151dc1abd41661302bb5fce2f810f7a5cd7dc